app/assets/stylesheets/getting-started.scss

```diff
@@ -1,7 +1,11 @@
 #hello-there {
-  .avatar {
-    width: 200px;
-    height: 200px;
+  .well {
+    padding-bottom: 30px;
+  }
```

Comment:
You could also fix the missing padding by updating `app/views/people/_add_contact.haml` to use correct Bootstrap 3 media objects. (See http://getbootstrap.com/components/#media and https://github.com/diaspora/diaspora/blob/develop/app/views/conversations/_message.haml)

For me that would be the preferred solution.